CONFIG.md

```diff
@@ -15,6 +15,8 @@ figma:
   lightFileId: shPilWnVdJfo10YF12345
   # [optional] Identifier of the file containing dark color palette and dark images.
   darkFileId: KfF6DnJTWHGZzC912345
+  # [optional] Figma API request timeout, default is 30s
```

Comment:
Add more info why a user may want to change the value. For example:
`# [optional] Figma API request timeout. The default value of this property is 30 (seconds). If you have a lot of resources to export set this value to 60 or more to give Figma API more time to prepare resources for exporting.`

Reply:
👍  I updated it.